public/javascripts/SVValidate/src/Tracker.js

```diff
@@ -71,6 +71,7 @@ function Tracker() {
             timestamp: timestamp,
             zoom: pov ? pov.zoom : null,
             is_mobile: isMobile()
+            // @mikey should this is_mobile be removed?
```

Comment:
@misaugstad can you confirm if this is_mobile should not be removed? Thanks!

Reply:
Is it being used anywhere that you haven't removed? Is there a reason that you're not sure if this one should be removed?